src/anet.c

```diff
@@ -491,10 +492,22 @@ int anetUnixServer(char *err, char *path, mode_t perm, int backlog)
     return s;
 }
 
+/* Accept a connection and also make sure the socket is non-blocking, and CLOEXEC.
+ * returns the new socket FD, or -1 on error. */
```

Comment:
I would put the comment on anetTCPAccept and anetUnixAccept since they are the ones that are expose into Redis, and have behavior changes.

Reply:
i thought about it and didn't wanna clone the comment.
since our comments are in the C file anyway, i thought that whoever comes looking for it, will find it here.
but if you still feel i better clone these two lines, let me know.

Reply:
We generally don't document functional contracts very well. It's very non-obvious that this function (as well as the two higher level functions) also set up the connection to be non-blocking and set CLOEXEC. I would probably put it  the documentation on all 3 so that it's clear, but it's not something I feel strongly about.

Reply:
I agree with @madolson here, better document the contract where it's used even if it's duplicate.